source/_data/MCPubs.bib

```diff
@@ -316,6 +319,13 @@ @Article{justitia:arxiv19
   biburl        = {https://dblp.org/rec/journals/corr/abs-1905-04437.bib},
   eprint        = {1905.04437},
   url           = {http://arxiv.org/abs/1905.04437},
+
+  publist_link = {paper || justitia-arxiv.pdf},
```

Comment:
Have you checked the link? The folder name must
match the bib key, otherwise the link won't work.